static/src/javascripts/projects/common/modules/experiments/ab.spec.ts

```diff
@@ -36,11 +36,15 @@ jest.mock('common/modules/experiments/ab-ophan', () => ({
 }));
 jest.mock('lodash/memoize', () => (f: (...args: unknown[]) => unknown) => f);
 
-const cfg = window.guardian.config as {
-	page?: Record<string, unknown>;
-	switches?: Record<string, boolean>;
+// from https://github.com/joonhocho/tsdef
```

Comment:
I can see how this is useful, but I found the definition of this type a bit convoluted.
I think it would be helpful to add the comment from Github to explain what this does, for readability:
`// make all properties optional recursively including nested objects`

Reply:
Fair enough. This builds on `Partial<T>`, [a TS utility type](https://www.typescriptlang.org/docs/handbook/utility-types.html#partialtype).